stdlib/2/md5.pyi

```diff
@@ -1,9 +1,16 @@
 # Stubs for Python 2.7 md5 stdlib module
 
+from typing import AnyStr
+
 class md5(object):
-    def update(self, arg: str) -> None: ...
-    def digest(self) -> str: ...
-    def hexdigest(self) -> str: ...
+    name = ...  # type: str
+    block_size = ...  # type: int
+    digest_size = ...  # type: int
+    digestsize = ...  # type: int
+    def __init__(self, arg: AnyStr= ...) -> None: ...
```

Comment:
Since you only use `AnyStr` once in the signature, it would be better to use `Union[str, unicode]`.